### DISCUSSION TITLE
<!--[if mso | IE]> This conditional check will only render the table in MSO or IE , but in my case it is applied in gmail ( chrome)

### DISCUSSION THREAD

**Discussion by Nishanth-sebastin** [mjmlio/mjml#2915] [Q&A] [answered]
I have this code in my generated html from mjml , but it should display in mso or IE, but in my case it is applied in gmail so i am having a issue in my mail

**Comment by iRyusa** [comment]
Try to send with somehting like https://putsmail.com/ to see if it comes from the way you're sending the email

**Reply by Nishanth-sebastin** [reply]
Thanks for the quick reply. even though I tried sending mail from putsmail by using my generated html , it works correctly and 
 In mjml playground also it worked correctly. but when i send a mail from my app , then i receive that kinda issue in my gmail.

**Reply by iRyusa** [reply]
Then your app is altering the HTML, there's nothing we can do about this.

**Reply by Nishanth-sebastin** [reply]
I’m not altering the HTML—it's generated directly from mjml2html. It works in the MJML Playground and Putsmail, but Gmail renders it differently when sent from my app. Any idea what could cause this

**Reply by iRyusa** [reply]
It comes from the way you send the HTML from your app. As said we can't do anything about that.

**Reply by Nishanth-sebastin** [reply, marked as answer]
Thank you! I resolved my issue, which was caused by logic that incorrectly handled in auto fix broken HTML logic.
